### ISSUE TITLE
PHP: Windows build for 1.20.0RC1 failed

### ISSUE THREAD

**Issue by stanley-cheung** [grpc/grpc#18667]
Relevant logs:
```
php_grpc.obj : error LNK2019: unresolved external symbol setenv referenced in function zm_activate_grpc
C:\php-snap-build\obj\7.1-nts-windows-vc14-x64\Release\php_grpc.dll : fatal error LNK1120: 1 unresolved externals
NMAKE : fatal error U1077: '"C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\BIN\amd64\link.exe"' : return code '0x460'
Stop.
```

The `setenv` call was introduced recently in #18539

**Comment by stanley-cheung** [comment]
Looks like we should be using `putenv()` instead.

**Comment by stanley-cheung** [comment]
Fixed in 1.20.0RC3
